package.json

```diff
@@ -4,18 +4,21 @@
   "version": "1.4.0",
   "jsdelivr": "dist/search-insights.min.js",
   "main": "dist/search-insights.cjs.min.js",
+  "types": "types/lib/node.d.ts",
   "engines": {
     "node": ">=8.16.0"
   },
   "files": [
     "dist",
     "lib",
+    "types",
     "!**/__tests__/**"
   ],
   "scripts": {
-    "build": "rollup --environment NODE_ENV:'production' -c rollup.config.js",
-    "build:dev": "rollup --watch --environment NODE_ENV:'development' -c rollup.config.js",
+    "build": "yarn build:types && rollup --environment NODE_ENV:'production' -c rollup.config.js",
```

Comment:
@adambullmer I tried `yarn run build` on my computer, and it didn't generate `types` folder. Can you look into this?

Reply:
This will actually output the types into the dist directory, colocated with their corresponding JS files. This was necessary to fix relative linking in the definition outputs, and for definition resolution.